### DISCUSSION TITLE
업스케일링 배율을 가져올 방법이 있나요

### DISCUSSION THREAD

**Discussion by vwyoungbokwi** [naverz/zepeto-studio-kor#1070] [5. Q&A]
UI 사이즈를 맞추려고 하는데 태블릿에서는 UI가 업스케일링되버려서 Constant Physical Size로 설정해도 크기를 맞출 수가 없습니다.

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 업스케일링 이슈를 파악해 볼 코드나 스크린샷을 첨부해주실 수 있을까요?

**Reply by vwyoungbokwi** [reply]
![1792](https://user-images.githubusercontent.com/108778243/181730407-0508d0f9-330d-4fff-8646-9730266155c6.jpg)
맨아래 아이폰처럼 제페토 UI위치 바로 옆에 저희 UI를 배치하려고 Canvas Scaler를 Constant Physical Size로 설정했는데 핸드폰 종류에 따라 조금씩 어긋나거나 아얘 2배정도 커지는 경우도 있습니다.
(사진에는 없지만, 아이패드에서 테스트했을때도 가운데처럼 제페토UI보다 2배정도 컸습니다.)

UI 설정값은 아래 프로젝트 파일과 같습니다.
[TestProject.zip](https://github.com/naverz/zepeto-studio-kor/files/9218760/TestProject.zip)

* 제페토 UI 옆에 UI를 배치하기위한 어떤 방법이든 알려주시면 감사하겠습니다!

**Reply by Zepeto-TechEvangelist** [reply]
안녕하세요, 모바일에서 일반적으로 사용하는 Scale with screen 설정을 적용해 보시는 것을 추천합니다.
이렇게 할 경우 해상도에 비례해서 동일한 위치에 UI가 생성됩니다. 
다만 기기마다 Safe Area가 다르기 때문에 이를 감안해서 채팅 버튼 바로 옆에 하는 것보다, 조금 넉넉하게 잡으시는 것이 좋습니다.

![스크린샷 2022-07-29 오후 8 20 29](https://user-images.githubusercontent.com/100118650/181748592-0c991962-4687-4b08-bce7-ff1665a30d70.png)
